int32 robot_id
float64 x
float64 y
float64 w
int32 items
